docs/oauth2.md

```diff
@@ -229,6 +171,50 @@ auth_oauth2.algorithms.1 = HS256
 auth_oauth2.algorithms.2 = RS256
 ```
 
+Each `auth_oauth2.resource_servers.<id/index>.` entry has the following settings:
+
+* `id`
+* `resource_server_type`
+* `additional_scopes_key`  
+* `scope_prefix`  
+* `preferred_username_claims`
+* `oauth_provider_id`
+
+Typically, a numeric value is used as `index`, e.g.  `auth_oauth2.resource_servers.1.id = rabbit_prod`. However, it can be any string, e.g. `auth_oauth2.resource_servers.rabbit_prod.jwks_url = http://some_url`. By default, the `index` is the resource server's id. However, you can override it via the `id` attribute like in `auth_oauth2.resource_servers.1.id = rabbit_prod`.
```

Comment:
I would use "Usually" instead of "Typically"
e.g. should be "For example, " GLOBAL COMMENT